app/src/main/java/com/simple/checkstyledemo/MainActivity.java

```diff
@@ -16,9 +16,7 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     private void test(int a) {
-        if (a > 3){
-            return;
-        }
+        if (a > 3)return;
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck
**Severity**: ERROR
**File**: app/src/main/java/com/simple/checkstyledemo/MainActivity.java L19

&apos;return&apos; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*